app/pb_kits/playbook/pb_date_range_stacked/date_range_stacked.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+module Playbook
+  module PbDateRangeStacked
+    class DateRangeStacked < Playbook::PbKit::Base
+      include ActionView::Helpers::TagHelper
+      include ActionView::Context
+
+      PROPS = %i[configured_classname
+                 configured_data
+                 configured_id
+                 configured_start_date
+                 configured_end_date
+                 configured_dark].freeze
+
+      def initialize(classname: default_configuration,
+                     data: default_configuration,
+                     id: default_configuration,
+                     start_date: default_configuration,
+                     end_date: default_configuration,
+                     dark: default_configuration)
+        self.configured_classname = classname
+        self.configured_data = data
+        self.configured_id = id
+        self.configured_start_date = start_date
+        self.configured_end_date = end_date
+        self.configured_dark = dark
+      end
+
+      def start_date
+        Playbook::PbKit::PbDateTime.new(configured_start_date)
+      end
+
+      def start_day_month
+        day_month = "#{start_date.to_day} #{start_date.to_month}"
+        pb_title = Playbook::PbTitle::Title.new(text: day_month, size: 4, dark: dark)
+        ApplicationController.renderer.render(partial: pb_title, as: :object)
```

Comment:
Besides moving rails kits to the "modernized" format, we are also removing all direct invocations of `ActionController`. There's no need to update this kit to the more recent convention, but it would be good if you could move this into the `html.erb` template itself. So instead of:

```html
<div class="pb_date_range_stacked_start_date_wrapper">
  <%= object.start_day_month %>
  <%= object.start_year %>
</div>
```

it'd be 

```html
<div class="pb_date_range_stacked_start_date_wrapper">
  <%= pb_rails("title", props: { text: object.day_month, size: 4, dark: object.dark }) %>
  <%= pb_rails("body", props: { text: object.start_year, color: light, dark: object.dark }) %>
</div>
```
and in your object file, these methods would be removed and replaced with:

```ruby
def day_month
  "#{start_date.to_day} #{start_date.to_month}"
end

def start_year
  start_date.to_year
end
```

Reply:
It also makes what's happening in the template more obvious.